### DISCUSSION TITLE
Is it possible to draw polygon like this and crop it accordingly?

### DISCUSSION THREAD

**Discussion by AlbertLiu1974** [CropperBlazor/Cropper.Blazor#325] [Ideas]
Hi @MaxymGorn  Kindly reference the image below:

![image](https://github.com/CropperBlazor/Cropper.Blazor/assets/152601781/f6790216-5517-4110-8bab-dc49f13b6d87)

Not sure if it's possible but it's very necessary for me to fulfill some request. Thanks a lot.  or some direction to help deal with it?

Br, Albert

**Comment by MaxymGorn** [comment]
Hi @AlbertLiu1974. Sorry, I was overwhelmed in this week and possibly next week and I can't change library for support third-party canvas drawing. You can change manually from img to canvas html tag in this code: https://github.com/CropperBlazor/Cropper.Blazor/blob/dev/src/Cropper.Blazor/Cropper.Blazor/Components/CropperComponent.razor#L10 if you need it changes urgently.  But be careful, canvas can't support @attributes or @onerror or @onload or another events/parameters for canvas html tag.

For cropping using polygon, you can use following tutorial: https://cropperblazor.github.io/examples/cropping#crop-a-polygon-image
